### PR TITLE
replace docker installation to binary

### PR DESCRIPTION
**Pull request by Yaiba**
- change the installation to binary instead of docker
- use same template for `kwil-cli` `kwild` `kwil-admin` installation page

### CODE REVIEW

File: docs/daemon/install.mdx [hunk 1]
```diff
@@ -7,11 +7,30 @@ description: How to install the Kwil Daemon tool
...
-While Kwil is under heavy development, the team has made the project closed source. However, the Kwil team regularly releases
-Kwil Docker images to Docker Hub. This allows you to run Kwil on your own machine, without having to build the project from source.
+Until the main Kwil DB repository is open source, the `kwild` command line 
+application may be downloaded from the Kwil binary release repository.
 
-To install the most recent Kwil image, run:
+
+## Download the `kwild` Executable
+
+The Kwil daemon(`kwild`) will be packaged together with Kwil admin tool(`kwil-admin`) and Kwil cli(`kwil-cli`).
```
Comment (jchappelow) [inline docs/daemon/install.mdx:16]: Nit, but when parenthesizing, a space between the preceding word/phrase and the parenthesis the norm.

File: docs/admin/install.mdx [hunk 2]
```diff
@@ -8,28 +8,13 @@ slug: /admin/installation
...
-```bash
-docker exec -it <container-name-or-id> /bin/sh
-```
-
-## Download the Standalone `kwil-admin` Executable
-
-The Kwil daemon (`kwild`) and the node administration tool will be
-packaged together. Download the [latest release](https://github.com/kwilteam/binary-releases/releases/tag/v0.6.0-beta)
+The Kwil admin tool(`kwil-admin`) will be packaged together with Kwil daemon (`kwild`) and  Kwil cli(`kwil-cli`). 
+Download the [latest release](https://github.com/kwilteam/binary-releases/releases/tag/v0.6.0-beta)
```
Comment (jchappelow) [inline docs/admin/install.mdx:17]: We *could* do "https://github.com/kwilteam/binary-releases/releases/latest" with "latest" so we don't have to update docs as frequently.

Comment (Yaiba) [reply docs/admin/install.mdx:17]: Yeah I want to change to latest too, should we do it?

Comment (KwilLuke) [reply docs/admin/install.mdx:17]: agree latest would be better

File: docs/kwil-cli/installation.md [hunk 3]
```diff
@@ -7,14 +7,30 @@ description: Install kwil-cli
...
 
-There are binaries for the following platforms:
 
-* Linux amd64
-* Linux arm64
-* Darwin (MacOS) Universal (amd64+arm64)
+## Download the `kwil-cli` Executable
 
-After downloading the binary for your platform, add the binary file path to the PATH environment variable.
+The Kwil cli(`kwil-cli`) will be packaged together with Kwil deamon(`kwild`) and Kwil admin tool(`kwil-admin`).
```
Comment (KwilLuke) [inline docs/kwil-cli/installation.md]: typo: Kwil Daemon*

File: docs/kwil-cli/installation.md [hunk 4]
```diff
@@ -7,14 +7,30 @@ description: Install kwil-cli
...
 
-There are binaries for the following platforms:
 
-* Linux amd64
-* Linux arm64
-* Darwin (MacOS) Universal (amd64+arm64)
+## Download the `kwil-cli` Executable
 
-After downloading the binary for your platform, add the binary file path to the PATH environment variable.
+The Kwil cli (`kwil-cli`) will be packaged together with Kwil deamon (`kwild`) and Kwil admin tool (`kwil-admin`).
```
Comment (KwilLuke) [inline docs/kwil-cli/installation.md:16]: Typo: Kwil Daemon*

**Pull request merged**